### DISCUSSION TITLE
Validate postcode/zipcode

### DISCUSSION THREAD

**Discussion by vuon9** [go-playground/validator#751] [Ideas]
Hi all,

I saw the list of regex for many kinds of post code format base on the country (XML)
https://github.com/unicode-org/cldr/blob/release-26-0-1/common/supplemental/postalCodeData.xml
or (PHP) https://gist.github.com/andresgcarmona/10ee75ec1cc31a71063974305f7f91a9

Stack Overflow zipcode regex related topic: https://stackoverflow.com/a/7185241/1155318

- Does it make sense to make it happens in this library?
- What is/are the rule(s) will be used in struct tag? E.g: `postcode_by_country_field=CountryCode`, `postcode_by_country=US`

Hope to hear about the opinion and the way to do if possible. Many thanks!

**Comment by deankarn** [comment]
Hey @vuon9 yes it's totally possible and think it could make sense to add them to this library.

My only thought would be I think they should be separate validations, one for each country, if all were to be added. That way you could combine or create aliases for them as needed.

**Reply by vuon9** [reply]
Ah, got your idea!
